### ISSUE TITLE
Breast Cancer Prediction using RNN

### ISSUE THREAD

**Issue by vanshu25** [srimani-programmer/Breast-Cancer-Predictor#111]

**Comment by srimani-programmer** [comment]
@vanshu25 please add a proper description of the issue.

**Comment by vanshu25** [comment]
@srimani-programmer Sir, I want to train a deep neural network with the given dataset to predict the breast cancer type. By using RNN, an efficient model can be prepared which will use effective features and can produce a result with high accuracy.

**Comment by srimani-programmer** [comment]
@vanshu25 issue is assigned to you.

**Comment by ainy-123** [comment]
@vanshu25 Are you still working on this issue. If no, @srimani-programmer please assign this to me .Hoping for speedy reply .
Thanks!
I am GSSoC'21 participant and would like to contribute to this issue.

**Comment by vanshu25** [comment]
@ainy-123 Yes, I am working on this issue, in fact, I have already completed it and have pulled a PR for the same.

**Comment by ainy-123** [comment]
> @ainy-123 Yes, I am working on this issue, in fact, I have already completed it and have pulled a PR for the same.

Thanks for the update.

**Comment by yash623623** [comment]
@srimani-programmer I would like to work on this issue, Please assign it to me

**Comment by Venkatakrishnan-Ramesh** [comment]
i would like to work on this issue using RNN

**Comment by navyasharma0203** [comment]
I would like to work on this issue. Can you assign this to me?
